app/routes/users/components/UserConfirmation.tsx

```diff
@@ -89,91 +95,94 @@ const UserConfirmation = ({
     <Container>
       <div className={styles.root}>
         <h2>Registrer bruker</h2>
-        <Form onSubmit={handleSubmit}>
-          <Field
-            name="username"
-            placeholder="Brukernavn"
-            label="Brukernavn"
-            component={TextInput.Field}
-          />
-          <PasswordField user={user} />
-          <Field
-            label="Passord (gjenta)"
-            name="retypePassword"
-            type="password"
-            autocomplete="new-password"
-            component={TextInput.Field}
-          />
-          <Field
-            name="firstName"
-            placeholder="Fornavn"
-            label="Fornavn"
-            autocomplete="given-name additional-name"
-            component={TextInput.Field}
-          />
-          <Field
-            name="lastName"
-            label="Etternavn"
-            placeholder="Etternavn"
-            autocomplete="family-name"
-            component={TextInput.Field}
-          />
-          <MultiSelectGroup label="Kjønn" name="gender">
-            <Field
-              name="genderMan"
-              label="Mann"
-              component={RadioButton.Field}
-              inputValue="male"
-            />
-            <Field
-              name="genderWoman"
-              label="Kvinne"
-              component={RadioButton.Field}
-              inputValue="female"
-            />
-            <Field
-              name="genderOther"
-              label="Annet"
-              component={RadioButton.Field}
-              inputValue="other"
-            />
-          </MultiSelectGroup>
-          <Field
-            name="allergies"
-            placeholder="Allergier"
-            label="Matallergier/preferanser"
-            component={TextInput.Field}
-          />
+        <LegoFinalForm onSubmit={onSubmit} validate={validate}>
+          {({ handleSubmit }) => {
+            return (
```

Comment:
A bigger fan of just letting it be a implicit return
```suggestion
          {({ handleSubmit }) => (
```